### PR TITLE
Updates smaller admin ship

### PR DESCRIPTION
**Pull request by Arokha**
Not as huge as the GARGANTUAN BEAST Ascian made (no I'll never stop poking fun at). Also removes all the old areas and unnecessary things to clean up the tether files.

ALSO fixes loading new z-levels with lighting.

I guess this is fluff?

### CODE REVIEW

**Comment by Heroman3003** [comment]
Was backup of tether10colony left in intentionally?

File: maps/tether/tether-10-colony.dmm [hunk 1]
```diff
@@ -25273,7 +25273,7 @@
 /area/unknown/dorm3)
 "WI" = (
 /turf/simulated/floor/wood,
-/area/houseboat)
+/area/space)
```
Comment (Novacat) [inline maps/tether/tether-10-colony.dmm]: I have a feeling that, given this is a simulated turf, it is not supposed to be area/space.

Comment (Arokha) [reply maps/tether/tether-10-colony.dmm]: Good point, but it's YOUR ship that it's used on :p

image

I'll fixy.

**Comment by Arokha** [comment]
> Was backup of tether10colony left in intentionally?

Nope, removed. We should probably add those backup folders to our gitignore

**Pull request merged**